package ru.practicum.explore_with_me.main.exception;

public class DataErrorException extends RuntimeException {

    public DataErrorException(String message) {
        super(message);
    }

}
